app/src/test/java/rick_and_morty/rules/FakeRealmProvier.kt

```diff
@@ -1,36 +1,60 @@
 package rick_and_morty.rules
 
 import io.realm.RealmObject
+import rick_and_morty.data.model.CharacterResultsDto
+import rick_and_morty.data.model.RealmCharacters
 import rick_and_morty.data.model.episodes.EpisodeResultDto
 import rick_and_morty.data.model.episodes.realm.RealmEpisodes
 import rick_and_morty.data.realm.RealmInstance
+import rick_and_morty.ui.characters.CharactersMapper.toRealmCharacter
 import rick_and_morty.ui.episodes.EpisodesMapper.toRealmEpisode
 
 class FakeRealmInstance : RealmInstance {
```

Comment:
For some reason the file name was not changed, when we went for `FakeRealmInstance`.

Reply:
Done